index.php

```diff
@@ -59,50 +57,28 @@
                     case 'CtoC':
-                        echo '<span class="error">Please select different units!</span>';
-                        break;
-                    case 'CtoF':
-                        $myTemp = celsius_to_fahrenheit($_POST['Val']);
-                        echo '<p class="temp">The temperature is ' . $myTemp . ' degrees Fahrenheit.</p>';
-                        // echo '<span >Convert C to F</span>';
-                        break;
-                    case 'CtoK':
-                        $myTemp = celsius_to_kelvin($_POST['Val']);
-                        echo '<p class="temp">The temperature is ' . $myTemp . ' Kelvins.</p>';
-                        //echo '<span >Convert C to K</span>';
-                        break;
-                    case 'FtoC':
-                        $myTemp = fahrenheit_to_celsius($_POST['Val']);
-                        echo '<p class="temp">The temperature is ' . $myTemp . ' degrees Celsius.</p>';
-                        //echo '<span>Convert F to C</span>';           
-                        break;
                     case 'FtoF':
-                        echo '<span class="error">Please select different units!</span>';
-                        break;
-                    case 'FtoK':
-                        $myTemp = fahrenheit_to_kelvin($_POST['Val']);
-                        echo '<p class="temp">The temperature is ' . $myTemp . ' Kelvins.</p>';
-                        //echo '<span>Convert F to K</span>';
-                        break;
-                    case 'KtoC':
-                        $myTemp = kelvin_to_celsius($_POST['Val']);
-                        echo '<p class="temp">The temperature is ' . $myTemp . ' degrees Celsius.</p>';
-                        // echo '<span >Convert K to C</span>';
-                        break;
-                    case 'KtoF':
-                        $myTemp = kelvin_to_fahrenheit($_POST['Val']);
-                        echo '<p class="temp">The temperature is ' . $myTemp . ' degrees Fahrenheit.</p>';
-                        // echo '<span \>Convert K to F</span>';
-                        break;
                     case 'KtoK':
                         echo '<span class="error">Please select different units!</span>';
                         break;
-                }
+                    default:
+                        $tmpUnit = array(
+                            "F"=>"Fahrenheit", 
+                            "C"=>"Celsius", 
+                            "K"=>"Kelvin"
+                        );
+                        $result = new TemperatureConversion();
+                        $result->set_Temperature($_POST['Val']);
+                        $result->set_ConvertType($type);                        
+                        $myTemp = $result->get_convertResult();
+                        echo '<p class="temp">The temperature is ' . $myTemp . ' degrees '.$tmpUnit[$_POST['selectTo']].'.</p>'; 
```

Comment:
Kelvin units are not referred to as degrees. There needs to be an additional line to reflect these results, like this one:
echo '<p class="temp">The temperature is ' . $myTemp . ' Kelvins.</p>';